src/locales/fr.json

```diff
@@ -205,6 +205,10 @@
       "title": "Nom d'usage",
       "label": "Votre nom d'usage est utilisé lorsque vous partagez des fichiers avec d'autres utilisateurs de Twake."
     },
+    "matrix_id": {
+      "title": "Matrix ID",
+      "label": "Matrix ID est un identifiant unique dans la plateforme Twake, utilisé pour la messagerie en temps réel dans le protocole open-source Matrix."
```

Comment:
nit: This sentence feels weird. I would either remove the comma or split it into to distinct sentence.